### ISSUE TITLE
incorrect values in results

### ISSUE THREAD

**Issue by robfitzgerald** [NREL/routee-compass#154]
after making most of the fixes in this milestone (and using the branch in #153 to test), i'm still seeing an invalid `traversal_result` for the denver test case:

```python
    "traversal_summary": {
        "battery_state": 48.35810155022617,    # SOC, initial == 100%!
        "distance": 0.0,                       # miles
        "energy_electric": 30.98513906986422,  # kWh
        "energy_liquid": 0.0, 
        "time": 46.931571471291974             # minutes!
    },
```

- we shouldn't wipe out half of the battery
- distance is missing from summary
- energy + time are super high

**Comment by nreinicke** [comment]
What config are you using to run this? I might try to replicate the results on my side.

**Comment by robfitzgerald** [comment]
sure thing, i used
  - config: routee-compass-tomtom/configs/compass_tomtom_denver_energy_smartcore.toml 
  - query: routee-compass-tomtom/queries/test_denver.json

**Comment by robfitzgerald** [comment]
> I think the missing distance will be fixed by #145

Ohhhh yaaaaaaa

**Comment by nreinicke** [comment]
Looking a little deeper into the huge energy consumption, I think some of it has to do with our grade values. The new dataset uses millis as a grade unit but our configs all indicate that the incoming grade is in decimal units. We also expect the grade unit to come in as `graph_grade_unit` but the configs are using `grade_table_grade_unit` and so those should be consistent

**Comment by robfitzgerald** [comment]
Closed by #157.
